src/oop/creating_objects_from_classes/hard/q2/MainTest.java

```diff
@@ -0,0 +1,71 @@
+package oop.creating_objects_from_classes.hard.q2;
```

Comment:
For this one, we are actually going to need to change it. This is more of an object interactions style question, but it should actually be about manipulating initializing and manipulating the object

Reply:
I am currently working on an example of this, and I will send it to you once I finish it